src/platform_impl/android/mod.rs

```diff
@@ -757,6 +757,12 @@ impl Window {
 
     pub fn set_max_inner_size(&self, _: Option<Size>) {}
 
+    pub fn resize_increments(&self) -> Option<PhysicalSize<u32>> {
+        None
+    }
+
+    pub fn set_resize_increments(&self, _increments: Option<Size>) {}
```

Comment:
Could you add `warn!("Setting resize increments is not supported")`? Though, look at the similar messages we are using, might need lowercase it.

Reply:
In those new dummy methods for other platforms I replicated surrounding comments/warnings/whatever - so that they look consistent with everything else that's already there